bench/wtperf/runners/500m-btree-50r50u.wtperf

```diff
@@ -1,19 +1,21 @@
-# wtperf options file: simulate riak and its test3 configuration
-# The configuration for the connection and table are from riak and the
-# specification of the data (count, size, threads) is from basho_bench.
+# wtperf options file: simulate MongoDB.
+# The configuration for the connection and table are from mongoDB.
+# We use multiple tables to simulate collections and indexes.
 # This test assumes that its correlating populate already completed and exists.
 #
-#conn_config="cache_size=21G,checkpoint_sync=false,mmap=false,session_max=1024,statistics=(fast,clear),statistics_log=(wait=60)"
-conn_config="cache_size=21G,checkpoint_sync=false,mmap=false,session_max=1024,log=(enabled),statistics=(fast),statistics_log=(wait=30)"
-checkpoint_interval=60
+# Set cache to half of memory.  Enable logging and checkpoints.  Collect
+# wiredtiger stats for ftdc.
+conn_config="cache_size=16G,checkpoint=(wait=60,log_size=2GB),session_max=1024,log=(enabled),statistics=(fast),statistics_log=(wait=30,json)"
```

Comment:
I'd prefer an eviction worker configuration in here.

It would also be good to add a table_config that sets page sizes explicitly